InOut/CMakeLists.txt

```diff
@@ -113,15 +105,15 @@ if(USE_PULSEAUDIO)
     make_plugin(rtpulse rtpulse.c ${PULSEAUDIO_LIBRARY} ${PULSESIMPLE_LIBRARY})
 endif()
 
-if(PORTAUDIO_FOUND OR USE_VCPKG)
+find_package(portaudio)
+if(USE_PORTAUDIO AND ((TARGET portaudio_static) OR (TARGET portaudio)))
```

Comment:
This syntax doesn't look great to me. I don't think this is the recommended way of doing things. Can you point to anywhere that says otherwise?

Reply:
Ref https://github.com/csound/csound/pull/1762#discussion_r1376936235